taxes_by_invoice/taxes_by_invoice.py

```diff
@@ -32,7 +30,7 @@ class PrintTaxesByInvoiceAndPeriodStart(ModelView):
             },
         domain=[
             ('fiscalyear', '=', Eval('fiscalyear')),
-            ])
+            ], help="Left empty to select all periods")
```

Comment:
I know the message was already like this but it is wrong. Should read "Leave empty...", not "Left empty..."